custom-plugin-decorators/remove-unused-tags/tags.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+  id: "tags",
+  decorators: {
+    oas3: {
+      "no-unused-tags": ({ ignore }) => {
+        console.log("Checking for unused tags...");
+        return {
+          Root: {
+            // need the whole document for this
+            leave(target, ctx) {
+              // case-insensitive so use lowercase
+              let tagsToIgnore = [];
+              if (ignore instanceof Array) {
+                for (i = 0; i < ignore.length; i++) {
+                  tagsToIgnore.push(ignore[i].toLowerCase());
+                }
+              }
+
```

Comment:
```suggestion
              const tagsToIgnore = ignore?.map(tag => tag.toLowerCase()) || [];
```

And I'd moved it up to the start of the decorator function.